### ISSUE TITLE
Tissue regions segmentations by QuPath annotation

### ISSUE THREAD

**Issue by asmagen** [trasse/OpSeF-IV#1]
Can this pipeline perform semantic segmentation of tissue regions such as for example tumor/normal/stromal/immune aggregates? Can it use the corresponding annotations I'm making in QuPath for a few images to be used as a test set and then apply the pixel classification into other slides? And does it utilize different resolutions of the image such that it can recognize both macro and micro structures such as general big tumor vs normal structures as well as smaller more local classes I'm defining within the tumor?

**Comment by trasse** [comment]
Q:
Can this pipeline perform semantic segmentation of tissue regions such as for example tumor/normal/stromal/immune aggregates? 

A:
This is in principle possible, but not with currently integrated pre-trained model.

Q:
Can it use the corresponding annotations I'm making in QuPath for a few images to be used as a test set and then apply the pixel classification into other slides? 

A:
The combination of OpSeF & QuPath would be very powerful for tissue data. I will reach out to Pete. QuPath is evolving very fast. I think the OpSeF & QuPath interface will be ideally based on the next stable release QuPath 0.2.0.

Q:
And does it utilize different resolutions of the image such that it can recognize both macro and micro structures such as general big tumor vs normal structures as well as smaller more local classes I'm defining within the tumor?

A:
This can be easily implemented. It would just be two independent runs, which you would combine later. E.g. nucleus segmentation & tumor detection.

**Comment by asmagen** [comment]
@trasse Thanks. Any updates or anticipated timeline here?
